packages/node/src/node.ts

```diff
@@ -175,13 +175,13 @@ export class Node {
       const fromXpub = this.publicIdentifier;
       const to = data.toXpub;
 
-      this.requestHandler.outgoing.emit(data.eventName, data);
-
       await this.messagingService.send(to, {
         data,
         from: fromXpub,
         type: NODE_EVENTS.PROTOCOL_MESSAGE_EVENT
       } as NodeMessageWrappedFinMessage);
+
+      this.requestHandler.outgoing.emit(data.eventName, data);
```

Comment:
How would this change prevent a potential race?

Reply:
Maybe race is the wrong word:

Right now, it seems like the bot farm responder shuts off when it hears the `installFinalEvent` which is *before* it's actually sending the final message to the initiator. Just trying this out to see if it helps fix the issue.